Index: editors/emacs30-nox11/version.mk
===================================================================
RCS file: /cvsroot/pkgsrc/editors/emacs30-nox11/version.mk,v
retrieving revision 1.1
diff -u -r1.1 version.mk
--- editors/emacs30-nox11/version.mk	16 Dec 2025 20:33:22 -0000	1.1
+++ editors/emacs30-nox11/version.mk	14 Aug 2026 18:06:24 -0000
@@ -1,7 +1,7 @@
 # $NetBSD: version.mk,v 1.1 2025/12/16 20:33:22 gdt Exp $
 
 _EMACS_FLAVOR=	emacs
-_EMACS_REQD=	emacs30-no-x11>=30.1<31
+_EMACS_REQD=	emacs30-nox11>=30.1<31
 
 _EMACS_VERSION_MAJOR=	30
 _EMACS_VERSION_MINOR=	2
