diff --git a/scripts/bootstrap/bootstrap.sh b/scripts/bootstrap/bootstrap.sh
index 58f985bdb2..26a32d5742 100755
--- a/scripts/bootstrap/bootstrap.sh
+++ b/scripts/bootstrap/bootstrap.sh
@@ -51,6 +51,8 @@ _BAZEL_ARGS="--spawn_strategy=standalone \
       --java_language_version=${JAVA_VERSION} \
       --tool_java_runtime_version=${JAVA_VERSION} \
       --tool_java_language_version=${JAVA_VERSION} \
+      --javacopt=-proc:full \
+      --host_javacopt=-proc:full \
       --define=protobuf_allow_msvc=true \
       ${DIST_BOOTSTRAP_ARGS:-} \
       ${EXTRA_BAZEL_ARGS:-}"
