--- a/src/s/mingw32.h
+++ b/src/s/mingw32.h
@@ -25,16 +25,13 @@
 
 #define MINGW
 
-/* We are using Cygwin-style headers in /usr/include, also used by
-   Cygwin itself */
-#define CYGWIN_HEADERS
 
 #ifndef ORDINARY_LINK
 #define ORDINARY_LINK
 #endif
 
-#define C_SWITCH_SYSTEM "-mno-cygwin -fno-caller-saves -DWIN32_NATIVE -DMINGW"
-#define LIBS_SYSTEM "-mno-cygwin -mwindows -lwinmm -lwsock32"
+#define C_SWITCH_SYSTEM "-fno-caller-saves -DWIN32_NATIVE -DMINGW"
+#define LIBS_SYSTEM "-mwindows -lwinmm -lwsock32"
 #define WIN32_LEAN_AND_MEAN
 
 #define TIME_ONESHOT 0
@@ -51,7 +48,7 @@
 /* #ifndef HAVE_SOCKETS */
 #define HAVE_SOCKETS
 /* #endif */
-#define OBJECTS_SYSTEM nt.o ntproc.o dired-msw.o
+#define OBJECTS_SYSTEM nt.o dired-msw.o
 
 #undef MAIL_USE_SYSTEM_LOCK
 #define HAVE_MSW_C_DIRED
@@ -66,7 +63,7 @@
 
 #ifndef NOT_C_CODE
 #include <stdlib.h>
-#include <../mingw/process.h>
+#include <../include/process.h>
 #define mkdir __mkdir
 #include <dir.h>
 #undef mkdir
--- a/src/sysproc.h
+++ b/src/sysproc.h
@@ -29,7 +29,7 @@
 #endif
 
 #ifdef MINGW
-#include <../mingw/process.h>
+#include <../include/process.h>
 #elif defined (WIN32_NATIVE)
 /* <process.h> should not conflict with "process.h", as per ANSI definition.
    This is not true with visual c though. The trick with <../include/process.h>
